crates/augurs-forecaster/src/transforms.rs

```diff
@@ -46,6 +53,8 @@ pub enum Transform {
     Logit,
     /// Log transform.
     Log,
+    /// Box-Cox transform.
+    BoxCox(f64),
```

Comment:
Since it's public, I would make this a struct variant to make it clear what the `f64` represents.

```suggestion
    BoxCox {
        lambda: f64,
    },
```